lib/engine/game/g_1849/game.rb

```diff
@@ -480,9 +517,13 @@ def operating_round(round_num)
             G1849::Step::Token,
             Engine::Step::Route,
             G1849::Step::Dividend,
+            G1849::Step::BuyToken,
+            G1849::Step::BuyEToken,
             Engine::Step::DiscardTrain,
             G1849::Step::BuyTrain,
             G1849::Step::IssueShares,
+            G1849::Step::BondInterestPayment,
+            G1849::Step::Bond,
             [Engine::Step::BuyCompany, { blocks: true }],
           ], round_num: round_num)
```

Comment:
Should we remove the unused steps here? At a minimum we shouldn't show in the log "skips bond" etc. if not enabled.

Reply:
Steps still here.

Reply:
I spoke with Chris; now I understand he meant to remove them _when_ they're unused (variant not enabled). Done with commit `variant steps are only included if respective variant is enabled.`